source/databases-collections.txt

```diff
@@ -119,31 +119,39 @@ initial collection.
 
 To create a new database:
 
-1. Hover over the connection for the deployment where you want your
+#. Hover over the connection for the deployment where you want your
    database to exist.
 
 #. Click the :icon-fa5:`plus` icon that appears.
 
-#. In the prompt, enter a name for your new database.
+#. A JavaScript file populates in a new tab with const 
```

Comment:
[suggestion]

Because this step isn't a step the user performs, maybe we should combine it with the previous step where we instruct users to click the plus icon. This way we're giving context on that step, but not making this seem like something the user needs to do.